client/genclients.go

```diff
@@ -176,16 +176,19 @@ func successResponse(outputName string) string {
 }
 
 func convertParamToString(p spec.Parameter) string {
-	capParamName := swagger.Capitalize(p.Name)
+	valToSet := fmt.Sprintf("i.%s", swagger.Capitalize(p.Name))
+	if !p.Required {
+		valToSet = "*" + valToSet
+	}
 	switch p.Type {
```

Comment:
might be useful in these switch statements to link to the swagger spec that enumerates the different values